images/build/go-runner/variants.yaml

```diff
@@ -1,11 +1,11 @@
 variants:
   go1.17-buster:
     CONFIG: 'go1.17-buster'
-    IMAGE_VERSION: 'v2.3.1-go1.17rc2-buster.0'
+    IMAGE_VERSION: 'v2.3.1-go1.17-buster.0'
```

Comment:
We can look at doing at `bullseye` variant once it's available for distroless: https://github.com/GoogleContainerTools/distroless/issues/780